src/Components/Common/input/input.jsx

```diff
@@ -0,0 +1,129 @@
+import React, { Component } from 'react';
+import Joi from 'joi-browser';
+// import { connect } from 'react-redux';
```

Comment:
Commented codes should not be present. These serve no purpose in the implementation of the project.